Swift/Tests/DictionaryTest.swift

```diff
@@ -88,7 +131,7 @@ class DictionaryTest: CBLTestCase {
         
         let level3 = MutableDictionaryObject()
         level3.setValue("n3", forKey: "name")
-        level2.setValue(level3, forKey: "level3")
+        level2.setDictionary(level3, forKey: "level3")
```

Comment:
This is not concistent as the rest is using setDictionary.

Reply:
You could convert this tests to all use setDictionary.

Reply:
I thought of ... but then there's no other test for setValue with a dict value, so I thought interchanging them would prove they behave the same and as expected.

Reply:
Then it’s better to have different tests to test them both. Objective-C test has a large number of tests that test all getters and setters. You may check and design what should be the best and optimized test cases for swift.

Reply:
One idea would be for specific tests that test dictionary, we can use dictionary methods. Then we can have a spec test to test get/set value method.